src/components/CardHead.js

```diff
@@ -19,24 +28,74 @@ export default function CardHead({
   handleDelete,
   id,
   pathname,
+  setProfiles,
+  profiles,
 }) {
   const { on, toggle } = useToggle(false)
 
   return (
     <StyledHead onClick={toggle}>
       <StatusStyled>{status}</StatusStyled>
-      {on === false
-        ? pathname === '/profile-pool' && (
-            <DeleteSign onClick={() => handleDelete(id)}>
-              <img src={deleteIcon} alt="trashbox" />
-            </DeleteSign>
-          )
-        : ''}
-      <img src={img} alt="portrait" />
+      {on === false && pathname === '/profile-pool' && (
+        <DeleteButton onClick={() => handleDelete(id)}>
```

Comment:
I would create for DeleteButton an own file where you handle the component. Then your code in this file is a bit shorter and cleaner.

Reply:
That was my idea in the first place but thats easier said than done. I tried it already and couldn't manage it. My app crashed. Piers finally told me not to do it. The problem is that the delete component needs to be in the component cardhead as when the card is toggled the delete button is not shown. If you have an idea how to manage it feel free to let me know : )